libs/agno/agno/vectordb/chroma/chromadb.py

```diff
@@ -197,31 +199,30 @@ def search(self, query: str, limit: int = 5, filters: Optional[Dict[str, Any]] =
         result: QueryResult = self._collection.query(
             query_embeddings=query_embedding,
             n_results=limit,
+            include=["metadatas", "documents", "embeddings", "distances", "uris"],
         )
 
         # Build search results
         search_results: List[Document] = []
 
         ids = result.get("ids", [[]])[0]
-        metadata = result.get("metadatas", [[]])[0]  # type: ignore
+        metadata = result.get("metadatas", [{}])[0]  # type: ignore
         documents = result.get("documents", [[]])[0]  # type: ignore
-        embeddings = result.get("embeddings")
+        embeddings = result.get("embeddings")[0]
+        embeddings = [e.tolist() if hasattr(e, "tolist") else e for e in embeddings]
         distances = result.get("distances", [[]])[0]  # type: ignore
-        uris = result.get("uris")
-        data = result.get("data")
-        metadata["distances"] = distances  # type: ignore
-        metadata["uris"] = uris  # type: ignore
-        metadata["data"] = data  # type: ignore
+        for idx, meta in enumerate(metadata):
+            meta["distances"] = distances[idx]  # type: ignore
```

Comment:
It makes more sense to me to loop through distances and assign to metadata with index? So just the other way around.